src/main/java/net/fabricmc/loader/transformer/accessWidener/AccessWidener.java

```diff
@@ -200,7 +200,7 @@ private static int makePublic(int i) {
 	}
 
 	private static int makeProtected(int i) {
-		if ((i & Opcodes.ACC_PRIVATE) != 0) {
+		if ((i & ~Opcodes.ACC_PUBLIC) == 0) {
```

Comment:
The method could still be package private...

You need `if((i & Opcodes.ACC_PUBLIC) != 0)`

Reply:
ah yeah, thanks :D